src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -229,229 +218,98 @@ where
 
 type WorkerId = usize;
 
-/// Reads from a [`SourceReader`] and returns a collection timestamped with the
+/// Reads from a [`SourceConnection`] and returns a collection timestamped with the
```

Comment:
Something like "Renders the source dataflow fragment from the given [`SourceConnection`]. This returns a collection ..." seems better.

Also, the name `source_reader_operator` is now outdated.

Reply:
Renamed to `source_render_operator`